releases/v2.1.0-alpha.20180730.md

```diff
@@ -0,0 +1,263 @@
+- The **Statements** page now shows statements that executed on all nodes in the cluster, not just the gateway node. [#26605][#26605] {% comment %}doc{% endcomment %}
+- The **Statement Details** page now includes a table showing statistics broken down by which node was the gateway node. [#26605][#26605] {% comment %}doc{% endcomment %}
+
+### Bug Fixes
+
+- Fixed the ordering of columns in the [`cockroach node status`](../v2.1/view-node-details.html) output. [#27042][#27042]
+- Fixed a bug that would make the **Statement Details** page in the Web UI break if a statement wasn't found. [#27105][#27105]
+- Fixed some incorrectly typed columns in the `pg_index` virtual table. [#27723][#27723]
+- Fixed permissions and audit logging issues with the optimizer. [#27108][#27108]
+- Prevented a situation in which ranges repeatedly fail to perform a split. [#26934][#26934]
+- Fixed a crash that could occur when distributed `LIMIT` queries were run on a cluster with at least one unhealthy node. [#26950][#26950]
+- Failed [`IMPORT`](../v2.1/import.html)s now begin to clean up partially imported data immediately and in a faster manner. [#26959][#26959]
+- `IMPORT` now detects node failure and will restart instead of fail. [#26881][#26881]
+- Fixed a panic in the optimizer with `IN` filters. [#27053][#27053]
+- Fixed a panic that could occur when renaming a scalar function used as a data source. [#27039][#27039]
+- The server will not finalize a version upgrade automatically and erroneously if there are nodes temporarily inactive in the cluster. [#26821][#26821] {% comment %}doc{% endcomment %}
+- The `DISTINCT ON` clause is now reported properly in statement statistics. [#27221][#27221]
+- Fixed a panic in [`IMPORT`](../v2.0/import.html) when creating a table using a sequence operation (e.g., `nextval()`) in a column's [DEFAULT](../v2.0/default-value.html) expression.  [#27122][#27122]
+- `SET` now properly rejects attempts to use
+  invalid variable names starting with `tracing.`. [#27216][#27216]
+- Fixed `NULL` equality handling in the experimental lookup join feature. [#27336][#27336]
+- `ALTER ... EXPERIMENTAL CONFIGURE ZONE` is now properly tracked in statement statistics. [#27213][#27213]
+- Invalid uses of set-generating functions in `FROM` clauses are now reported with the same error code as PostgreSQL. [#27390][#27390]
+- The number of `COPY` columns is now correctly verified during `IMPORT ... PGDUMP`. [#27345][#27345]
+- `CHANGFEED`s now correctly emit all versions of quickly changing rows. [#27612][#27612]
+- Alleviated a scenario in which a large number of uncommitted Raft commands could cause memory pressure at startup time. [#27009][#27009]
+- Prevented the unbounded growth of the Raft log caused by a loss of quorum. [#27774][#27774]
+- Foreign key references in `IMPORT ... PGDUMP` are now processed in the correct order. [#27782][#27782]
+- Fix the "global matching" regex flag to `g`. [#27128][#27128] {% comment %}doc{% endcomment %}
```

Comment:
This last release note is unreadable and does not make sense.

This is also a doc change, not a bug fix. The proper release note is:

```
Release note (doc change): the online documentation for the `g` flag accepted by the
SQL built-in function `regexp_replace()` has been corrected.
```

Reply:
Removed. Thanks.